CMP1903_A1_2324/Die.cs

```diff
@@ -8,16 +8,19 @@ namespace CMP1903_A1_2324
 {
     internal class Die
     {
-        /*
-         * The Die class should contain one property to hold the current die value,
-         * and one method that rolls the die, returns and integer and takes no parameters.
-         */
-
-        //Property
-
-
-        //Method
+        private Random random;
+        public int dieRolled {get; private set;}
+        public Die()
+        {
+            random = new Random();
+            dieRolled = 0;
+        }
```

Comment:
Not quite sure if this is necessary, it would create the same effect if line 11 were changed to private static Random random = new Random(); and this method was removed.